### ISSUE TITLE
Is it possible to try using mse instead of cross entropy as the loss function?

### ISSUE THREAD

**Issue by SujayTalanki** [karpathy/nanoGPT#397]
Currently, the model uses F.cross_entropy as the loss function. I was wondering if we could use F.mse_loss as the loss function, and what the hyperparameters to that function would look like?

![image](https://github.com/karpathy/nanoGPT/assets/91350869/b194ca1e-585e-4b4b-8145-ccc59cd1bd0f)

I was wondering how this block would change. Thank you!

**Comment by shendiaomo** [comment]
I don't think so. We have no meaningful labels in a language model for calculating mean squared error.

**Comment by VatsaDev** [comment]
@SujayTalanki try looking into xVal, they used mse
